LoopKit/CarbKit/CarbStore.swift

```diff
@@ -28,23 +28,11 @@ public protocol CarbStoreDelegate: class {
 
 public protocol CarbStoreSyncDelegate: class {
 
-    /// Asks the delegate to upload recently-added carb entries not yet marked as uploaded.
-    ///
-    /// The completion handler must be called in all circumstances with each entry passed to the delegate
-    ///
-    /// - parameter carbStore:  The store instance
-    /// - parameter entries:    The carb entries
-    /// - parameter completion: The closure to execute when the upload attempt(s) have completed. The closure takes a single argument of an array of entries. Populate `externalID` and set `isUploaded` for each entry that was uploaded, or pass back the entry unmodified for each entry that failed to upload.
-    func carbStore(_ carbStore: CarbStore, hasEntriesNeedingUpload entries: [StoredCarbEntry], completion: @escaping (_ entries: [StoredCarbEntry]) -> Void)
+    /// Ask the delegate to initiate remote data synchronization. Remote data synchronization is an asynchronous
+    /// process that pulls data from multiple local sources and synchronizes with any remote data services. The process
+    /// can take considerable time and no completion handler is provided.
+    func initiateRemoteDataSynchronization()
```

Comment:
I think something like `func carbStoreDidUpdate(_ carbStore: CarbStore)` might be better here.

Reply:
Done.